test/ginkgo/Makefile

```
@@ -0,0 +1,108 @@
+# Tools - assuming they are in the path or in the project's bin directory
+KUSTOMIZE ?= $(CURDIR)/../../bin/kustomize
+KUBECTL ?= kubectl
+K3D ?= k3d
+
+K3D_CLUSTER_NAME ?= test-e2e-local
+
+ifndef ignore-not-found
+  ignore-not-found = false
+endif
+
+.PHONY: deploy-argocd-operator kustomize test-e2e-local k3d-cluster-create k3d-cluster-delete k3d-image-import
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)